CharlieBackend.Data/ApplicationContext.cs

```diff
@@ -39,6 +39,10 @@ public ApplicationContext(DbContextOptions<ApplicationContext> options)
 
         public virtual DbSet<Attachment> Attachments { get; set; }
 
+        public virtual DbSet<Hometask> Hometasks { get; set; }
+
+        public virtual DbSet<Hometask> AttachmentOfHometask { get; set; }
```

Comment:
DbSet&lt;Hometask&gt; AttachmentOfHometask
to  DbSet<**AttachmentOfHomework**> AttachmentsOfHomework